Packs/DNSDB/Integrations/DNSDB/DNSDB.yml

```diff
@@ -0,0 +1,320 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: DNSDB
+  version: -1
+configuration:
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: DNSDB Service URL
+  name: url
+  defaultvalue: https://api.dnsdb.info/
+  type: 0
+  required: false
```

Comment:
Why would we change this? url is an optional parameter.

Reply:
> The default value is nice but can be deleted, it should be mandatory.

Very few will change this from the default. A small number of customers run their own DNSDB API servers but the vast majority use https://api.dnsdb.info. We would prefer not to incur the support overhead of making this a mandatory option.

Reply:
I am not sure I followed. Not having it as a mandatory field will cause more support issues. In practice, a user can (even accidentally) delete this value, and since it is not mandatory he could still save the configuration, the next thing would be that every command will crash as there is no URL configured. Having a default value is indeed designed for cases like this where you want to have a pre-determined value that will probably serve most of the users, however, it still doesn't solve the issue I explained here above.